backend/python/app/services/interfaces/story_service.py

```diff
@@ -50,3 +50,14 @@ def get_story_translations(self, user_id, translate):
         :rtype: list of StoryTranslationResponseDTO's
         """
         pass
+
+    @abstractmethod
+    def get_story_available_for_review(self, level, language):
```

Comment:
nit: can you order params like the `get_stories_available_for_translation` fn? i.e. language and then level